datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -1622,6 +1622,9 @@ private void configurePolicyResolvers(final RuntimeWiring.Builder builder) {
         })).dataFetcher("resolvedRoles", new LoadableTypeBatchResolver<>(dataHubRoleType, (env) -> {
             final ActorFilter filter = env.getSource();
             return filter.getRoles();
+        })).dataFetcher("resolvedOwnershipTypes", new LoadableTypeBatchResolver<>(ownershipType, (env) -> {
+            final ActorFilter filter = env.getSource();
+            return filter.getResourceOwnersTypesUrns();
```

Comment:
Can you confirm that we will resolve the entire "OwnershipType" entity from this? It appears that we are just mapping to the URNs themselves, as opposed to the OwnershipType objects that are required in the data model

Reply:
I think it does, how else would UI work properly? Here is query and the output screensho (please let me know if I misunderstood you)t:
![Screenshot from 2023-06-21 11-51-14](https://github.com/datahub-project/datahub/assets/1629032/06b73da9-dbb5-4016-a72f-05f4adf4422f)